src/processors/default/processPlatformTracks/processPlatformTracks.ts

```diff
@@ -18,9 +18,9 @@ const getNFTFactories = async (nfts: NFT[], dbClient: DBClient) => {
   if (nfts.length === 0) {
     throw new Error('Unexpected empty NFT array');
   }
-  const nftFactoryAddresses = _.uniq(nfts.map((n: NFT) => n.contractAddress));
-  const nftFactoryAddressesString = JSON.stringify(nftFactoryAddresses).replace(/\"/g, "'").replace('[', '(').replace(']', ')');
-  const query = `select * from "${Table.nftFactories}" where id in ${nftFactoryAddressesString}`
+  const nftFactoryIds = _.uniq(nfts.map((n: NFT) => n.nftFactoryId));
```

Comment:
do we migrate all  n.contractAddress =>  n.nftFactoryId properly to ensure they all are as expected?

Reply:
yes I did this in the migration. We rename contractAddress -> nftFactoryId, and then we update contractAddress to equal nftFactoryId. So everything will be updated as we expect